locations/spiders/addresses/address_spider.py

```diff
@@ -0,0 +1,29 @@
+from scrapy import Spider
+
+
+class AddressSpider(Spider):
```

Comment:
I'll have to move this out of `locations/spiders`, it can be done after the below so the suggestions work.